src/styles/_catalog.scss

```diff
@@ -1,22 +1,17 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  gap: 48px;
   margin: 0 auto;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
```

Comment:
This margin in no needed. Cause you use gap

Reply:
This is not fixed

Reply:
You didn't fix it 
![image](https://user-images.githubusercontent.com/99403198/203973097-524fbe4e-56d5-431c-ba54-4e96c55ffc51.png)